tpchgen-cli/Cargo.toml

```diff
@@ -9,6 +9,11 @@ homepage = { workspace = true }
 license = { workspace = true }
 repository = { workspace = true }
 
+[lib]
+name = "tpchgen_cli"
```

Comment:
this must match the python directory name (`tpchgen_cli`), see https://www.maturin.rs/project_layout#mixed-rustpython-project